docs/_embed/solutions/partner-api.md

```diff
@@ -349,14 +352,50 @@ Pragma: no-cache
   "refresh_token": "9D9oz2ZzaouT12LpvklQwNBf6s4vva"
 }
 ```
-🎉 This response contains the access token that you'll use to make API request on the user's behalf.
+🎉 This response contains the access token that you'll use to make API request on the user's behalf, as well as a
+refresh token which should be stored securely for later use.
 
 ##### 5. Using the access token
-The access token should be passed with requests an `Authorization` header. For example:
+The access token should be passed with requests as an `Authorization` header. For example:
 ```
 Authorization: Bearer jk8s9dGJK39JKD93jkd03JD
 ```
 
+##### 6. Refreshing the access token
+All access tokens will expire after 10 hours (the number of seconds in `expires_in`), for security purposes. After
+that point, any request using that access token will return a 401 status code. To proceed, the refresh token should be
+exchanged for a new access token _and_ a new refresh token. This will not require any interaction by the user.
+
+Below is an example request that can be used:
+```cURL
+curl -v -u {CLIENT_ID}:{CLIENT_SECRET} \
+-d "grant_type=refresh_token&refresh_token={REFRESH_TOKEN}" \
+https://zapier.com/oauth/token/
+```
+* `CLIENT_ID` - This will be the same client id that you retrieved earlier.
+* `CLIENT_SECRET` - This is a secret known only to your application and the authorization server. It will be the same client secret that you retrieved earlier.
+* `REFRESH_TOKEN` - This is the refresh token code you received with the access token.
+
+You'll receive a response that looks like this:
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
+Pragma: no-cache
+
+{
+  "access_token": "NEfSRKpUjVd3Nj9yyaXKs15BrM7SVA",
+  "expires_in": 36000,
+  "token_type": "Bearer",
+  "scope": "zap zap:write authentication",
+  "refresh_token": "zzdumGAW2TmayeKjzu0z9oHJiziKdn"
+}
+```
+
+Note that you will receive a _new_ refresh token - the old refresh token can no longer be used again, and so the
+new refresh token should be stored securely for future use. Refresh tokens do have an expiration date - they only expire
```

Comment:
This sounds a little contradictory. Should it say that refresh tokens **don't** have an expiration date?

Reply:
Typo - thanks for catching that!